src/Storage/Query/Handler/LatestQueryHandler.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Storage\Query\Handler;
+
+use Bolt\Storage\Query\ContentQueryParser;
+use Bolt\Storage\Query\QueryResultset;
+
+/**
+ *  Handler to modify query based on activation of 'latest' modifier.
+ *
+ *  eg: 'pages/latest/10'
+ */
+class LatestQueryHandler
+{
+    public function __invoke(ContentQueryParser $contentQuery): QueryResultset
+    {
+        $contentQuery->setDirective('order', '-id');
+
+        return call_user_func($contentQuery->getHandler('select'), $contentQuery);
```

Comment:
return $contentQuery->getHandler('select')($contentQuery);

using call_user_func* is sooo PHP5